python/pyarrow/_compute.pxd

```diff
@@ -65,6 +65,8 @@ cdef CExpression _true
 
 cdef CFieldRef _ensure_field_ref(value) except *
 
+cdef vector[CSortKey] unwrap_sort_keys(sort_keys, allow_str=*)
```

Comment:
Should be `except *` since it calls `_ensure_field_ref` which can raise an exception.